vnext/Microsoft.ReactNative/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="boost" version="1.68.0.0" targetFramework="native" />
+  <package id="boost" version="1.72.0.0" targetFramework="native" />
   <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="native" />
-  <package id="Microsoft.Windows.CppWinRT" version="2.0.200316.3" targetFramework="native" />
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
```

Comment:
Do we have to bump cppwinrt as well? I recently discovered community modules are coupled to the cppwinrt version of the host app (which isn't great...). Bumping it has the potential to cause issues because of that.